src/courtroom.cpp

```diff
@@ -1305,13 +1305,14 @@ void Courtroom::done_received()
 
   if (char_list.size() > 0)
   {
-    set_char_select_page();
+    //set_char_select_page();
```

Comment:
?

Reply:
Caused a segfault, functionality still remains the same